Gems/ROS2/README.md

```diff
@@ -52,61 +24,34 @@ sudo apt install ros-${ROS_DISTRO}-ackermann-msgs ros-${ROS_DISTRO}-control-tool
 
 For a "feel" of these features, see an [example project](#example-project) which uses this Gem to run navigation stack.
 
-## Clone this repository
+## Example project
+
+You can test O3DE ROS2 Gem with the [Robot Vacuum](https://github.com/o3de/RobotVacuumSample) project. It will allow you to run robot navigation. All necessary assets are included.
```

Comment:
```suggestion
You can test the O3DE ROS2 Gem with the [Robot Vacuum](https://github.com/o3de/RobotVacuumSample) project. It will allow you to run a robot navigation simulation. All necessary assets are included.
```